ci/firebase_testlab.py

```diff
@@ -19,28 +19,28 @@
 
 
 def RunFirebaseTest(apk, results_dir):
-  try:
-    # game-loop tests are meant for OpenGL apps.
-    # This type of test will give the application a handle to a file, and
-    # we'll write the timeline JSON to that file.
-    # See https://firebase.google.com/docs/test-lab/android/game-loop
-    # Pixel 4. As of this commit, this is a highly available device in FTL.
-    subprocess.check_output([
-        'gcloud',
-        '--project', 'flutter-infra',
-        'firebase', 'test', 'android', 'run',
-        '--type', 'game-loop',
-        '--app', apk,
-        '--timeout', '2m',
-        '--results-bucket', bucket,
-        '--results-dir', results_dir,
-        '--device', 'model=flame,version=29',
-    ])
-  except subprocess.CalledProcessError as ex:
-    print(ex.output)
-    # Recipe will retry return codes from firebase that indicate an infra
-    # failure.
-    sys.exit(ex.returncode)
+  # game-loop tests are meant for OpenGL apps.
+  # This type of test will give the application a handle to a file, and
+  # we'll write the timeline JSON to that file.
+  # See https://firebase.google.com/docs/test-lab/android/game-loop
+  # Pixel 4. As of this commit, this is a highly available device in FTL.
+  process = subprocess.Popen(
+    [
+      'gcloud',
+      '--project', 'flutter-infra',
+      'firebase', 'test', 'android', 'run',
+      '--type', 'game-loop',
+      '--app', apk,
+      '--timeout', '2m',
+      '--results-bucket', bucket,
+      '--results-dir', results_dir,
+      '--device', 'model=flame,version=29',
+    ],
+    stdout=subprocess.PIPE,
+    stderr=subprocess.STDOUT,
+    universal_newlines=True,
+  )
+  return process
```

Comment:
Changes here are to run multiple firebase testlab tests in parallel. Now that we're running two this makes a difference. The new test executes pretty quickly but still requires setup/upload/results processing time.